domain/src/main/assets/image_region_selection_test_exploration.json

```diff
@@ -0,0 +1,177 @@
+{
```

Comment:
Not sure if you have already ran it so this is more of a reminder in case you haven't: Please format the file on https://jsonformatter.org/

Reply:
(Feel free to resolve this and merge if you have already done it)

Reply:
thanks for mentioning but yeah I did format this using above tool